mmv1/third_party/terraform/services/bigtable/resource_bigtable_instance.go

```diff
@@ -152,6 +152,13 @@ func ResourceBigtableInstance() *schema.Resource {
 							Computed:    true,
 							Description: `The state of the cluster`,
 						},
+						"node_scaling_factor": {
+							Type:         schema.TypeString,
```

Comment:
Instead of using diff.ForceNew function, `ForceNew:    true,` can be used for this field.

Reply:
Done